### ISSUE TITLE
Invalid tsdb database

### ISSUE THREAD

**Issue by kphowell** [delph-in/pydelphin#260]
(run_agg) [kphowell@patas pydelphin]$ delphin select "readings where i-wf==1" ../output/processed/abz/abz1
Traceback (most recent call last):
  File "/home2/kphowell/Envs/run_agg/bin/delphin", line 11, in <module>
    sys.exit(main())
  File "/home2/kphowell/Envs/run_agg/lib/python3.6/site-packages/delphin/main.py", line 42, in main
    args.func(args)
  File "/home2/kphowell/Envs/run_agg/lib/python3.6/site-packages/delphin/main.py", line 103, in call_select
    args.TESTSUITE)
  File "/home2/kphowell/Envs/run_agg/lib/python3.6/site-packages/delphin/commands.py", line 250, in select
    db = tsdb.Database(path, autocast=True)
  File "/home2/kphowell/Envs/run_agg/lib/python3.6/site-packages/delphin/tsdb.py", line 321, in __init__
    raise TSDBError('not a valid TSDB database: {!s}'.format(path))
delphin.tsdb.TSDBError: not a valid TSDB database: ../output/processed/abz/abz1

**Comment by goodmami** [comment]
@kphowell Thanks for the error message. Can you verify that `../output/processed/abz/abz1` is a directory and it contains a `relations` file? Those two conditions are how PyDelphin determines if a path is a valid profile.

**Comment by kphowell** [comment]
It is and it does. I just sent you a tarball with the profile via email. I'm going offline for the night and will be traveling a bit, but I'm happy to answer any other questions. Email me if you don't hear back.

**Comment by goodmami** [comment]
Ok thanks, Kristen, I got it.

I don't immediately see any problem with the profile. I don't get the error message you did, so perhaps your relative path above didn't point to the directory you think it did?

```console
$ delphin select 'i-input where readings = 0 & i-wf = 1' output/processed/abz/abz1/
Naana nenahaa hawel.
Ama nu hemayool difal laak.
Nepikaai kiding.
newiil nawel#
[...]
```

When you get a chance, let me know which version of PyDelphin you're using (`delphin -V`).

**Comment by kphowell** [comment]
HI Mike. It's 1.0.0.
It is possible I was pointing to the wrong place. I'm trying to repro it
now and I'm running into some problems with my python path. I'll need to
spend some more time on this when I'm back to work (early January). Since
you're able to use the profile, it's almost certainly an issue on my end.
Thanks for your help!

**Comment by goodmami** [comment]
@kphowell It's also possibly a Windows issue (if you're using it in Windows).

In any case, I recommend installing in a virtual environment (to help avoid those PYTHONPATH issues). Also, try updating to v1.0.3, as there was a bug I fixed with TSDB databases (#257). I don't think you're encountering this bug, but might as well use the newer version to be sure.

Also, a heads-up: I found an issue with TSQL queries that you'd encounter if you searched for the 'error' field (#262). It's now fixed in the `v1.1.0` branch and will make it into the next release.

**Comment by goodmami** [comment]
@kphowell I'm closing this with the v1.1.0 release as I cannot reproduce it. I fixed some issues related to TSDB and TSQL, so if after upgrading to v1.1.0 you still get the error, please re-open with any additional information that can help me narrow down the issue.
